google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpanTest.java

```diff
@@ -231,23 +227,27 @@ public void tearDown() {
 
   @Test
   public void singleUseNonRetryableErrorOnNext() {
-    expectedException.expect(SpannerMatchers.isSpannerException(ErrorCode.FAILED_PRECONDITION));
     try (ResultSet rs = client.singleUse().executeQuery(SELECT1AND2)) {
       mockSpanner.addException(FAILED_PRECONDITION);
       while (rs.next()) {
         // Just consume the result set.
+        fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done